opensrp-anc/build.gradle

```diff
@@ -189,7 +192,7 @@ dependencies {
         exclude group: 'org.yaml', module: 'snakeyaml'
         exclude group: 'io.ona.rdt-capture', module: 'lib'
     }
-    implementation('org.smartregister:opensrp-client-core:4.3.26-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-core:6.0.2-PREVIEW-SNAPSHOT@aar') {
```

Comment:
Are we using preview because client-core isn't merged yet? Please link the PR that needs merging for this to change.

Reply:
Yes, https://github.com/opensrp/opensrp-client-core/pull/871

Reply:
https://github.com/opensrp/opensrp-client-core/pull/871/files